src/clipboard.ts

```diff
@@ -72,17 +73,34 @@ export const copyToClipboard = async (
         }, {} as BinaryFiles)
       : undefined,
   };
-  const json = JSON.stringify(contents);
-  CLIPBOARD = json;
+
+  const clipboardDataAsText = transformClipboardElementsToText(
+    contents.elements,
+  );
+  CLIPBOARD = clipboardDataAsText;
+
   try {
     PREFER_APP_CLIPBOARD = false;
-    await copyTextToSystemClipboard(json);
+    await copyTextToSystemClipboard(clipboardDataAsText);
   } catch (error: any) {
     PREFER_APP_CLIPBOARD = true;
     console.error(error);
   }
 };
 
+export const transformClipboardElementsToText = (
+  elements: readonly NonDeletedExcalidrawElement[],
+): string => {
+  const onlyTextElements = elements.every((element) => element.type === "text");
+  if (onlyTextElements) {
+    return elements
+      .map((element) => normalizeText((element as ExcalidrawTextElement).text))
+      .join("\n");
```

Comment:
```suggestion
      .join("\n\n");
```
Separate individual elements on paragraphs (also aligns with the `copy as text` contextmenu option)